src/forms/StudyInfoForm/NewStudyForm.js

```diff
@@ -295,6 +296,7 @@ const NewStudyForm = ({
                   type="button"
                   loading={formikProps.isSubmitting}
```

Comment:
It seems like this `formikProps.isSubmitting` isn't working correctly because it never goes to a loading state for me. Perhaps change it to `submitting` instead?